BioSQL/BioSeq.py

```diff
@@ -229,22 +223,27 @@ def _retrieve_seq(adaptor, primary_id):
     # We have no way of knowing if these sequences will use IUPAC
     # alphabets, and we certainly can't assume they are unambiguous!
     if moltype == "dna":
-        alphabet = Alphabet.generic_dna
+        alphabet = "DNA"
```

Comment:
The mix of upper case (SeqRecord annotation) and lower case (values in BioSQL DB) is unavoidable, but in the context of the DBSeq object can't we just ignore alphabet now to match our Seq object?

i.e. Can't we set the DBSeq __init__ alphabet argument to None?

Reply:
I am in favor of that, but then we first need to fix the `__add__` method and friends on `Seq` and `MutableSeq` objects. Otherwise, without an `alphabet` attribute on a `DBSeq` object, we will not be able to add a `Seq` and a `DBSeq` object. Could you look into that? I would suggest to check whether the `other` argument in these functions is a `Seq`, `MutableSeq`, `UnknownSeq`, or string object. This will automatically capture `DBSeq` objects, as `DBSeq` is a subclass of `Seq`.

Reply:
We're thinking on the same lines here - had already started that for ``Seq``.